pyproject.toml

```
@@ -0,0 +1,104 @@
+[build-system]
+requires = ["setuptools>=61.0", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "pointpats"
+dynamic = ["version"]
+authors = [
+    { name = "Serge Rey", email = "sjsrey@gmail.com" },
+    { name = "Hu Shao", email = "shaohutiger@gmail.com" },
+]
+maintainers = [{ name = "pysal contributors" }]
+license = { text = "BSD 3-Clause" }
+description = "Methods and Functions for planar point pattern analysis"
+keywords = [
+    "spatial statistics, point patterns"
+]
+readme = { text = """\
+"Methods and Functions for planar point pattern analysis"
+
+""", content-type = "text/x-rst" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: GIS",
+]
+requires-python = ">=3.7"
```

```suggestion
requires-python = ">=3.8"
```

We should also drop `3.8`,  but that can be in another PR.

---

i wasnt sure how to set this. Is that what we chose for all the other packages? I figure the minimum supported version isnt the same thing as the minimum *required* version, and this will probably run on python 3.6+ if i had to guess, so didnt want to force a more recent version, even if its technically unsupported

basically im worried that if we set this too high, it will cause unnecessary install failures, when we should allow people to install unsupported versions if they want

---

We should just follow SPEC0. Then we don't need to have these discussions.